tests/explainers/computer_vision/test_explainers.py

```diff
@@ -148,6 +148,100 @@ def test_explainers_gpu(
         ) as xai_model:
             _, _ = xai_model(img_tensor)
 
+    @pytest.mark.parametrize("explainer_name", list(CVClassificationExplainers))
+    @pytest.mark.parametrize("batch_size", [4])
```

Comment:
I could use a range of batch sizes but it will significantly increase a unit test run. Or move it as a magic number inside the test suite.

Reply:
I meant, why list(), and [] ?

Reply:
So basically, `list(CVClassificationExplainers)` iterates over all possible Enum values and returns list of each possible value of Enum. That's why there is `list` keyword instead of, let's say `[CVClassificationExplainers]`.